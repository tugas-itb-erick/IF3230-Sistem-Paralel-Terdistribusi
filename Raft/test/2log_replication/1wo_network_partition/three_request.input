5
set_timeout 1 0
simulate 10
client_send_request 1 + 5
client_send_request 1 * 5
client_send_request 1 - 2
simulate 3
get_leader_state 1
get_state 1
get_state 2
get_state 3
get_state 4
get_state 5
simulate 3
get_leader_state 1
get_state 1
get_state 2
get_state 3
get_state 4
get_state 5
simulate 3
get_leader_state 1
get_state 1
get_state 2
get_state 3
get_state 4
get_state 5
simulate 3
get_leader_state 1
get_state 1
get_state 2
get_state 3
get_state 4
get_state 5
end_simulation
