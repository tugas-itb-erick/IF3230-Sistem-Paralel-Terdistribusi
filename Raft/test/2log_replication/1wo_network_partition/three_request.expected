L1 next_index[X 2 2 2 2 ] match_index[X 1 1 1 1 ]
S1 leader term:2 voted_for:1 commit_index:1 data:5 logs:T0[0] T2[+5*5-2]
S2 follower term:2 voted_for:1 commit_index:0 data:0 logs:T0[0] T2[+5]
S3 follower term:2 voted_for:1 commit_index:0 data:0 logs:T0[0] T2[+5]
S4 follower term:2 voted_for:1 commit_index:0 data:0 logs:T0[0] T2[+5]
S5 follower term:2 voted_for:1 commit_index:0 data:0 logs:T0[0] T2[+5]
L1 next_index[X 3 3 3 3 ] match_index[X 2 2 2 2 ]
S1 leader term:2 voted_for:1 commit_index:2 data:25 logs:T0[0] T2[+5*5-2]
S2 follower term:2 voted_for:1 commit_index:1 data:5 logs:T0[0] T2[+5*5]
S3 follower term:2 voted_for:1 commit_index:1 data:5 logs:T0[0] T2[+5*5]
S4 follower term:2 voted_for:1 commit_index:1 data:5 logs:T0[0] T2[+5*5]
S5 follower term:2 voted_for:1 commit_index:1 data:5 logs:T0[0] T2[+5*5]
L1 next_index[X 4 4 4 4 ] match_index[X 3 3 3 3 ]
S1 leader term:2 voted_for:1 commit_index:3 data:23 logs:T0[0] T2[+5*5-2]
S2 follower term:2 voted_for:1 commit_index:2 data:25 logs:T0[0] T2[+5*5-2]
S3 follower term:2 voted_for:1 commit_index:2 data:25 logs:T0[0] T2[+5*5-2]
S4 follower term:2 voted_for:1 commit_index:2 data:25 logs:T0[0] T2[+5*5-2]
S5 follower term:2 voted_for:1 commit_index:2 data:25 logs:T0[0] T2[+5*5-2]
L1 next_index[X 4 4 4 4 ] match_index[X 3 3 3 3 ]
S1 leader term:2 voted_for:1 commit_index:3 data:23 logs:T0[0] T2[+5*5-2]
S2 follower term:2 voted_for:1 commit_index:3 data:23 logs:T0[0] T2[+5*5-2]
S3 follower term:2 voted_for:1 commit_index:3 data:23 logs:T0[0] T2[+5*5-2]
S4 follower term:2 voted_for:1 commit_index:3 data:23 logs:T0[0] T2[+5*5-2]
S5 follower term:2 voted_for:1 commit_index:3 data:23 logs:T0[0] T2[+5*5-2]
